src/index.html

```diff
@@ -5,9 +5,33 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image"></div>
+      <div class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">Product code: 195434</div>
+      <div class="card__rate">
+        <div class="card__stars stars">
```

Comment:
```suggestion
        <div class="card__stars stars stars--4">
```

Reply:
Not fixed

Reply:
Not fixed

Reply:
<img width="595" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/3afdead4-223f-4ea9-9a10-c762cd329119">